chat/browser/03-Muxing-Encryption/README.md

```diff
@@ -1,5 +1,7 @@
 # Chapter 03 - Muxing and Encryption
 
+**Note**: Expect to have an AggregateError in console as your connection needs to be upgraded. See [#25](/../../issues/25) for more details.
+
```

Comment:
For Chapter 03, you shouldn't get an error once it's completed, it should connect just fine. Were you hitting errors here?

Reply:
My bad. Fixed it.